src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -119,12 +128,11 @@ const BottomSheetBackdropComponent = ({
       <Animated.View
         style={containerStyle}
         pointerEvents={pointerEvents}
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
+        accessibilityRole={_providedAccessibilityRole ?? undefined}
+        accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+        accessibilityHint={_providedAccessiblityHint ?? undefined}
+        {...rest}
```

Comment:
@Mahmoud-SK why is the spread of the `rest` props needed?

Reply:
It's needed to pass the rest of the properties to the Animated.View - in this case it's only animatedPosition-prop.